packages/wp-now/src/wp-now.ts

```diff
@@ -42,20 +46,23 @@ export default async function startWPNow(
 	options: Partial<WPNowOptions> = {}
 ): Promise<{ php: NodePHP; phpInstances: NodePHP[]; options: WPNowOptions }> {
 	const { documentRoot } = options;
-	const nodePHPOptions: PHPLoaderOptions = {
-		requestHandler: {
-			documentRoot,
-			absoluteUrl: options.absoluteUrl,
-		},
+	const nodePHPOptions: PHPRequestHandlerConfiguration = {
+		documentRoot,
+		absoluteUrl: options.absoluteUrl,
 	};
 
 	const phpInstances = [];
+	const loadRuntime = async () =>
+		await NodePHP.loadRuntime(options.phpVersion);
```

Comment:
> Property 'loadRuntime' does not exist on type 'typeof NodePHP'.

This GitHub message looks suspicious, otherwise the PR seems to look good.

Reply:
I think the GitHub message was because it was not using the latest version of Playground dependencies. Now it's gone.